### PR TITLE
feat: update ReadMe file

### PR DESCRIPTION
**Pull request by abypunnoose-2020MT93021**
## Changes

> **Concisely** explain what has changed in this PR. Does your PR require any other PRs to be merged first?  What does a reviewer need to look for? What context will they need?

### User Story / Defect

> **If applicable**, add a link to the user story or defect that this applies to. This will help your reviewer get context.

### CODE REVIEW

**Comment by VinayAdaki** [comment]
Please update the Description Section and remove the sections if not applicable.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,30 @@
 # OSSE-Assignment-4
 This is assignment 4 of Open Source Software Engineering.
+
+
+**Changes done by Aby : Start**
+
+Ide used to make project : Eclipse
```
Comment (VinayAdaki) [inline README.md]:
```suggestion
IDE used to make project : Eclipse
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+
+**Step followed**
+
+
```
Comment (VinayAdaki) [inline README.md]: if possible please remove extra line

File: README.md [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+    2. Create new branch ( locally)
+
+    3. Work on new branch ( change code base)
+
+    4. Push to remote branch
+
+    5. Create a PR and assign it to Vinay Adaki
+
+
+
```
Comment (VinayAdaki) [inline README.md]: above 2 extra lines can be removed

**Review by VinayAdaki — CHANGES_REQUESTED** [review]
Some comments provided

File: README.md (hunk 3 above)
Comment (abypunnoose) [reply README.md]: Sure, Will do that

File: README.md (hunk 2 above)
Comment (abypunnoose) [reply README.md]: on it.

**Review by abypunnoose — APPROVED** [review]

**Comment by abypunnoose** [comment]
Approving the changes

**Review by VinayAdaki — APPROVED** [review]

**Pull request merged**